meinberlin/templates/base.html

```diff
@@ -5,15 +5,37 @@
 <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>{% block title %}{% translate 'meinBerlin' %}{% endblock %}</title>
-    <link rel="shortcut icon" type="image/x-icon" href="{% static 'images/favicon.ico' %}?v=2" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Berlin.de favicon -->
+    <meta name="og:image" content="/static/images/Berlin_de_FB_Social.png" />
```

Comment:
@philli-m can you make use of the static tag instead? --> `"{% static 'images/...png' %}"`

Reply:
@m4ra not sure, this is just stuff just copied from their style guide, will try it out!

Reply:
@m4ra also I have rebased on dev and pushed as was going to change those on this branch but made sense to make a new one incase you are working on this branch you may want to pull